src/utils/getThresholdLib.ts

```diff
@@ -31,6 +31,8 @@ function getBitcoinConfig(): BitcoinConfig {
     client: shouldMockBitcoinClient ? new MockBitcoinClient() : undefined,
     network,
     credentials: !shouldMockBitcoinClient ? credentials : undefined,
+    // FIXME: It's a temporary workaround to get the connection working.
+    clientOptions: { rejectUnauthorize: false },
```

Comment:
We will need to set the options for `wss` only. So maybe we could reflect that in the code?
```suggestion
    clientOptions: credentials.protocol === "wss" ? { rejectUnauthorize: false } : undefined,
```

Reply:
71aff73bdb799a9522da89a53df93f87d11f8699